Source/GSThemeDrawing.m

```diff
@@ -2665,8 +2669,8 @@ - (void) drawScrollViewRect: (NSRect)rect
 	    {
               xpos = [vertScroller frame].origin.x + scrollerWidth;
 	    }
-          NSRectFill(NSMakeRect(xpos, [vertScroller frame].origin.y - 1.0, 
-                                1.0, scrollerHeight + 1.0));
+          NSRectFill(NSMakeRect(xpos, [vertScroller frame].origin.y, 
+                                1.0, scrollerHeight - (borderWidth * 2)));
```

Comment:
I had to look through the code of [NSScrollerView tile] for quite some time, but now I am sure what you are doing here is correct.
Maybe it could be simplified by reusing [horizScroller frame].size.width but I am unsure. There are so many different cases that get handled in the tile code.

Reply:
There's still one place where scrollers are drawn incorrectly - NSTableView. I guess magic numbers in original code appear for the sake of NSTableView drawing. I'll create a separate PR.